es6/jsxcad-ui-app.js

```diff
@@ -39809,7 +39881,9 @@ class App extends ReactDOM$2.Component {
 
               if (domElementByHash.has(entry.hash)) {
                 // Reuse the element we built earlier
-                console.log(`Re-appending ${entry.hash} to ${path}/${id}`);
+                console.log(`Re-appending ${entry.hash} to ${path}/${id}`); // FIX: This is a problem because it removes the child from where it
```

Comment:
I did have an issue where switching from view to edit mode would require re-running the notebook for the figures to appear sometimes